Dockerfile

```diff
@@ -160,6 +176,7 @@ CMD gunicorn \
 # in a single layer, shared between `lms-dev` and `cms-dev`.
 FROM base as dev
 RUN pip install -r requirements/edx/development.txt
+ENV LMS_CFG="$CONFIG_ROOT/lms.yml"
```

Comment:
was this a hold over from testing?

Reply:
No, we need to reset LMS_CFG after we change it for the docker-based target